### PR TITLE
[FEATURE] SCSS theme

### PR DESCRIPTION
**Pull request by fshovchko**
Common theme styles theme for all components

### CODE REVIEW

File: src/components/theme/theme.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+$base-color: #fff;
```
Comment (mtalstykh-exadel) [inline src/components/theme/theme.scss]: let's move it to the src/styles/theme.scss

File: src/components/login/loginForm/LoginForm.jsx [hunk 2]
```diff
@@ -82,19 +82,22 @@ const LoginForm = () => {
                 variant="outlined"
                 type="password"
                 name="password"
+                error={error || (errors.password && touched.email) ? true : false}
                 onChange={handleChange}
                 onBlur={handleBlur}
                 value={values.password}
               />
             </div>
+            {console.log(errors.email, errors.password)}
```
Comment (mtalstykh-exadel) [inline src/components/login/loginForm/LoginForm.jsx]: remove console log

File: src/store/actions/actionTypes.js [hunk 3]
```diff
@@ -11,3 +11,4 @@ export const LOGIN_FAILURE = 'LOGIN_FAILURE';
 export const LANGUAGE_CHANGE = 'LANGUAGE_CHANGE';
 
 export const LOGOUT_START = 'LOGOUT_START';
+export const THEME_CHANGE = 'HEADER_CHANGE';
```
Comment (mtalstykh-exadel) [inline src/store/actions/actionTypes.js]: why 'HEADER_CHANGE'?

Comment (fshovchko) [reply src/store/actions/actionTypes.js]: https://github.com/mtalstykh-exadel/team4_frontend/blob/387392c6a027c7ce8a29dad78800c40d1813ebec/src/store/actions/actionTypes.js#L14

File: src/components/login/loginForm/LoginForm.jsx (hunk 2 above)
Comment (fshovchko) [reply src/components/login/loginForm/LoginForm.jsx]: https://github.com/mtalstykh-exadel/team4_frontend/blob/45a529e8a4417de855d57f38123bea9757e307c6/src/components/login/loginForm/LoginForm.jsx#L89-L92

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Pull request merged**